controller/src/statemachine.rs

```diff
@@ -1,6 +1,12 @@
-use models::node::{BottlerocketShadow, BottlerocketShadowSpec, BottlerocketShadowState};
+use models::node::{
+    BottlerocketShadow, BottlerocketShadowSpec, BottlerocketShadowState, BottlerocketShadowStatus,
+};
 
+use chrono::Utc;
 use tracing::instrument;
+use tracing::{event, Level};
+
+const RETRY_MAX_DELAY_IN_MINUTES: i64 = 24 * 60;
```

Comment:
This should likewise be stored as a `Duration` to avoid having to encode the type in the name (`MINUTES`)

Reply:
I think we'd want to use `std::time::Duration`. It does seem to mostly rely on second granularity for its interface, but I think it's usually a good idea to encode a measured unit in your type, rather than in your variable name.

I think with the lack of minute interface, I'm willing to disagree but ship it in spite of this.